spec/definitions/SubscriptionSummaryMetrics.yaml

```diff
@@ -0,0 +1,19 @@
+type: object
+properties:
+  currency:
+    description: Currency (three letter ISO 4217 alpha code) (eg USD, EUR)
+    readOnly: true
+    type: string
+  invoiced:
+    description: "Total amount invoiced. Formatted according to the currency, example: $23.02"
+    readOnly: true
+    type: string
+  collected:
+    description: "Total amount paid. Formatted according to the currency, example: $15.85"
+    readOnly: true
+    type: string
+  numberOfInvoices:
+    description: Number of active invoices
```

Comment:
active?

Reply:
I am filtering out `draft`, `voided` and `abandoned` statuses so I wanted a way to describe that. I suppose it is misleading language? What do you think?

Reply:
Maybe:
> Number of issued, past due, or paid invoices